setup.py

```diff
@@ -32,9 +32,16 @@
     pass
 
 
-requires = ['satpy!=0.38.*,!=0.39.*,!=0.40.*,!=0.41.*', 'pyorbital', 'trollsift', 'pyspectral', 'h5netcdf']
-
-
+install_requires = [
+    "h5netcdf",
+    "pyorbital",
+    "pyspectral",
+    "trollsift",
+    "satpy[avhrr_l1b_eps,viirs_l1b,viirs_sdr,viirs_compact]!=0.38.*,!=0.39.*,!=0.40.*,!=0.41.*",
+]
+extras_requires = {
+    "extra": "satpy[avhrr_l1b_gaclac,seviri_l1b_hrit,seviri_l1b_native,seviri_l1b_nc,vii_l1b_nc]"
```

Comment:
note that it is possible to separate this further if desired, e.g.:

```
extras_requires = {
     "gaclac": "satpy[avhrr_l1b_gaclac]",
     "seviri": "satpy[seviri_l1b_hrit,seviri_l1b_native,seviri_l1b_nc]",
     "metimage": "satpy[vii_l1b_nc]",
}

Reply:
I think the further specification is good.

Reply:
I added more otpions.